app/src/main/java/co/icanteach/projectx/util/BaseViewModel.kt

```diff
@@ -0,0 +1,21 @@
+package co.icanteach.projectx.util
+
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Job
+import kotlin.coroutines.CoroutineContext
+
+abstract class BaseViewModel(
```

Comment:
Sure, it's possible. Although for unit test I've found some issues. So when i'm going to do some test, I rather use this `BaseViewModel` instead of using `viewModelScope`